main.rb

```diff
@@ -0,0 +1,89 @@
+require_relative 'app'
+
+def list_option
+  puts ''
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+end
+
+def create_person(app)
+  puts 'Do you want to create a student(1) or a teacher(2)? [Input the number]: '
+  option = gets.chomp
+
+  parent_permission = true
+
+  puts 'age:'
+  age = gets.chomp.to_i
+
+  puts 'name: '
+  name = gets.chomp
+
+  puts 'Has parent permission? [Y/N]'
+  permission_value = gets.chomp
+
+  parent_permission = false if %w[N n NO no No].include?(permission_value)
+
+  case option.to_i
+
+  when 1
+    app.create_student(name, age, parent_permission)
+
+  when 2
+    puts 'Specialization: '
+    specialization = gets.chomp
+    app.create_teacher(name, age, parent_permission, specialization)
+  else
+    puts 'Invalid input :( '
+  end
+end
```

Comment:
Great work so far 👍🏼 , I would just suggets that you keep all app entry points inside app class to keep your work organized , this function can be merged with create student and create teacher as this last onces are just creating new objects